front-end-tools/src/Main.tsx

```diff
@@ -588,13 +660,29 @@ export default function Main(props: ConnectionProps) {
                                                             ? moduleReferenceDeployed
                                                             : moduleReferenceCalculated
                                                     );
-                                                    setInitName(contracts[0]);
+
+                                                    setDisplayContracts(contracts);
+                                                    setContractName(contracts[0]);
                                                 }
                                             }}
                                         />
                                         <span>{' Use Module from Step 1'}</span>
                                     </label>
                                 </div>
+                                {useModuleFromStep1 && (
+                                    <>
+                                        <br />
+                                        <div>
+                                            This checkbox autofilled the `moduleReference`, the `contractNames`, and the
+                                            `embeddedInputParameterSchema` from the above module.
```

Comment:
I don't think this backtick works very well. It looks ugly. Perhaps use https://getbootstrap.com/docs/4.0/content/code/#inline-code instead?
![Screenshot from 2023-07-11 20-15-13](https://github.com/Concordium/concordium-smart-contract-tools/assets/105212/ba1b8986-d74a-42ad-ad4b-bf962b871da3)